docs/pages/workflow/expo-cli.md

```diff
@@ -5,1083 +5,284 @@ maxHeadingDepth: 4
-| `--config [file]`              | Deprecated: Use app.config.js to switch config files instead. |
-
-</p>
-</details>
-
-<details>
-<summary>
-<h4>expo fetch:ios:certs</h4>
-<p>Download the project's iOS standalone app signing credentials</p>
-</summary>
-<p>
-
-| Option            | Description                                                   |
-| ----------------- | ------------------------------------------------------------- |
-| `--config [file]` | Deprecated: Use app.config.js to switch config files instead. |
-
-</p>
-</details>
-
-<details>
-<summary>
-<h4>expo fetch:android:keystore</h4>
-<p>Download the project's Android keystore</p>
-</summary>
-<p>
-
-| Option            | Description                                                   |
-| ----------------- | ------------------------------------------------------------- |
-| `--config [file]` | Deprecated: Use app.config.js to switch config files instead. |
-
-</p>
-</details>
-
-<details>
-<summary>
-<h4>expo fetch:android:hashes</h4>
-<p>Compute and log the project's Android key hashes</p>
-</summary>
-<p>
-
-| Option            | Description                                                   |
-| ----------------- | ------------------------------------------------------------- |
-| `--config [file]` | Deprecated: Use app.config.js to switch config files instead. |
-
-</p>
-</details>
-
-<details>
-<summary>
-<h4>expo fetch:android:upload-cert</h4>
-<p>Download the project's Android keystore</p>
-</summary>
-<p>
-
-| Option            | Description                                                   |
-| ----------------- | ------------------------------------------------------------- |
-| `--config [file]` | Deprecated: Use app.config.js to switch config files instead. |
-
-</p>
-</details>
+`expo run:ios` can only be run on a Mac, and Xcode must be installed. You can build the app in the cloud from any computer using `eas build -p ios`. Likewise `expo run:android` requires Android Studio and Java to be installed and configured on your computer.
 
----
+Building locally is useful for developing native modules, and debugging complex native issues. Building remotely with `eas build` is often much more resilient due to the preconfigured environment.
 
-### Notifications
+If your project does not have the corresponding native folder, the prebuild will be run one-time to generate the folder before building. For example, if your project does not have an `ios/` folder in the root directory, then running `npx expo run:ios` will first run `npx expo prebuild -p ios` before building your app. This folder can be treated as a temporary folder, and added to your project's `.gitignore` to remain in the "managed workflow". The native project's can be regenerated with `npx expo prebuild --clean` at any time. Because the iOS build cache (AKA "derived data") folder lives outside of your project, you can `--clean` while retaining relatively fast rebuild times.
```

Comment:
```suggestion
If your project does not have the corresponding native folder, the prebuild will run once to generate the folder before building. For example, if your project does not have an `ios/` folder in the root directory, then running `npx expo run:ios` will first run `npx expo prebuild -p ios` before building your app. This folder can be treated as a temporary folder, and you can add it to your project's `.gitignore` to remain in the managed workflow. The native project can be regenerated with `npx expo prebuild --clean` at any time. Because the iOS build cache (also known as "derived data") folder lives outside of your project, you can `--clean` while retaining relatively fast rebuild times.
```